base/c9s-python-3.9/Dockerfile

```
@@ -0,0 +1,87 @@
+FROM quay.io/sclorg/s2i-base-c9s:c9s
```

```suggestion
FROM quay.io/sclorg/python-39-c9s:c9s
```

the upstream https://github.com/sclorg/s2i-python-container/blob/master/3.9/Dockerfile.c9s is building the same base image.
probably we can utilize the upstream and extend on top of it ?
quay.io/sclorg/python-39-c9s:c9s

this would reduce our overhead of maintaining the base image

---

Good catch! Thanks for the suggestion. I pushed the new changes.